dotCMS/src/main/java/com/dotcms/rest/api/v1/index/ESIndexResource.java

```diff
@@ -480,44 +440,106 @@ public Response getActive(@Context HttpServletRequest httpServletRequest, @Conte
         }
     }
 
-    @GET
-    @Path("/docscount/{params:.*}")
-    @Produces("text/plain")
-    public Response getDocumentCount(@Context HttpServletRequest httpServletRequest,
-            @Context final HttpServletResponse httpServletResponse,
-            final @PathParam("params") String params)
-            throws DotSecurityException {
 
-        InitDataObject init= null;
-        try {
-            init = auth(params,httpServletRequest, httpServletResponse);
-
-            //Creating an utility response object
-            ResourceResponse responseResource = new ResourceResponse( init.getParamsMap() );
-
-            String indexName = this.indexHelper.getIndexNameOrAlias(init.getParamsMap(),"index","alias",this.indexAPI);
-            return responseResource.response( Long.toString( indexDocumentCount( indexName ) ) );
-
-        } catch (DotDataException e) {
-            Logger.error(this.getClass(),"Exception trying to get document count: " + e.getMessage(), e);
-            return ResponseUtil.mapExceptionResponse(e);
-        }
-    }
 
     @GET
     @Path("/indexlist/{params:.*}")
     @Produces(MediaType.APPLICATION_JSON)
     public Response indexList(@Context HttpServletRequest httpServletRequest, @Context final HttpServletResponse httpServletResponse, @PathParam("params") String params) {
         try {
-            InitDataObject init = auth(params,httpServletRequest, httpServletResponse);
+            InitDataObject init = auth(httpServletRequest, httpServletResponse);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)